genrules/BUILD

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 EngFlow Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# NOTE
+# The following genrules allow for an intensive test on the building system
```

Comment:
```suggestion
# The following genrules are able to produce thousands of targets based on the user's input.
# This can be used for stress testing your build infrastructure.
```